definitions/component-mapping/component-mapping.yaml

```diff
@@ -36,3 +47,27 @@ spec:
         #!/usr/bin/env bash
         mapping=$(curl $(params.mapping-file) | yq -o=json -I=0)
         echo ${mapping} | tee $(results.mapping.path)
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apply-mapping
+spec:
+  params:
+    - name: application-snapshot
+      type: string
+      description: The ApplicationSnapshot to apply the mapping to
+    - name: mapping
+      type: string
+      description: The mapping to apply to the snapshot
+  results:
+    - name: filtered-snapshot
+      description: The filtered AppicationSnapshot of images
+  steps:
+    - name: apply-mapping
+      image: quay.io/hacbs-release/release-utils
+      script: |
+        #!/usr/bin/env bash
+        curl -O -L https://raw.githubusercontent.com/johnbieren/release-service/test_mapping_file/filter.py
```

Comment:
This script could be moved somewhere, just not sure where yet

Reply:
A repo in the release-service organization.

Reply:
That's fine for now. We are talking about the possibility of using `jq` to do the same thing. Not for this milestone though.

Reply:
I moved the script in this PR https://github.com/hacbs-release/strategy-configs/pull/1. Once it is merged I will update this line. I added the `do-not-merge` label to this PR in the meantime